{
  "comments": [
    {
      "key": {
        "uuid": "02e03151_262d7c55",
        "filename": "appengine/monorail/api/converters.py",
        "patchSetId": 1
      },
      "lineNbr": 379,
      "author": {
        "id": 1167909
      },
      "writtenOn": "2018-07-24T00:04:04Z",
      "side": 1,
      "message": "Should IngestApprovalDelta specify this too?\nAlso, just curious, where is this exception raised exactly?",
      "range": {
        "startLine": 379,
        "startChar": 38,
        "endLine": 379,
        "endChar": 63
      },
      "revId": "123ab315e19617746f08a291b856ae0701be1e09",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    }
  ]
}